minerl/Malmo/Minecraft/build.gradle

```diff
@@ -346,7 +346,38 @@ task copyModToServer(type: Copy) {
 // Interesting. We've packaged forge gradle into the full jar and use gradle inherently??
 shadowJar {
     classifier = 'fat'
-    configurations = [project.configurations.all]
+    configurations = [
```

Comment:
where does this list of configurations come from?

Reply:
I updated this to be more clean